src/runtime/pkg/katautils/config.go

```diff
@@ -440,6 +445,15 @@ func (h hypervisor) getTxRateLimiterCfg() (uint64, error) {
 	return h.TxRateLimiterMaxRate, nil
 }
 
+func (h hypervisor) getGuestMemoryDumpFormat() string {
+	switch h.GuestMemoryDumpFormat {
+	case "elf", "kdump-zlib", "kdump-snappy", "kdump-lzo":
```

Comment:
The config file specifies ` kdump-lazo` rather than ` kdump-lzo` so which one is correct?